packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -42,4 +72,70 @@ describe('Error', () => {
         // Assert
         expect(description.text()).toMatchSnapshot();
     });
+
+    describe('methods', () => {
+        describe('`redirectToMenu` ::', () => {
+            const oldWindowLocation = window.location;
+
+            beforeAll(() => {
+                delete window.location;
+
+                window.location = Object.defineProperties(
+                    {},
+                    {
+                        ...Object.getOwnPropertyDescriptors(oldWindowLocation),
+                        assign: {
+                            configurable: true,
+                            value: jest.fn()
+                        }
+                    }
+                );
+            });
+
+            afterAll(() => {
+                window.location = oldWindowLocation;
+            });
+
+            it('should exist', () => {
+                expect(wrapper.vm.redirectToMenu).toBeDefined();
+            });
```

Comment:
I think we discussed this with some of the team members and we decided not to add these tests anymore, since we don't think it adds any value. If a method/computed is not defined, all the associated tests will fail.

Reply:
Disagree but can remove. I think having the tests fail as early as possible has the advantage of being able why a test fails in a more simpler format (it doesn't exist) rather than a more complex failure further down the chain.

Reply:
Done.